src/components/Checkbox/Checkbox.js

```diff
@@ -111,35 +114,42 @@ const CheckboxInput = styled('input')`
 
 const CheckboxLabel = styled('label')`
   ${labelBaseStyles}
-  ${labelCheckedStyles}
   ${labelDisabledStyles}
   ${labelInvalidStyles}
 `;
 
+const CheckboxWrapper = styled('div')`
+  ${checkboxWrapperBaseStyles};
+`;
+
 /**
  * Checkbox component for forms.
  */
-const Checkbox = ({ onToggle, children, id: customId, ...props }) => {
+const Checkbox = ({ onChange, children, id: customId, ...props }) => {
   const id = customId || uniqueId('checkbox_');
   return (
-    <Fragment>
-      <CheckboxInput id={id} onClick={onToggle} type="checkbox" {...props} />
+    <CheckboxWrapper>
+      <CheckboxInput id={id} onClick={onChange} type="checkbox" {...props} />
       <CheckboxLabel htmlFor={id} {...props}>
         {children}
       </CheckboxLabel>
-    </Fragment>
+    </CheckboxWrapper>
   );
 };
 
 Checkbox.propTypes = {
   /**
    * Controles/Toggles the checked state.
    */
-  onToggle: PropTypes.func.isRequired,
+  onChange: PropTypes.func.isRequired,
   /**
    * Value string for input.
    */
-  value: PropTypes.string.isRequired,
+  value: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.bool,
+    PropTypes.array
+  ]),
   /**
    * Child nodes to be rendered as the label.
    */
```

Comment:
Do you mind changing the `radio group` to `checkbox group` two lines below?

Reply:
Sorry I cannot figure out what you're referring to. Can you take a screenshot?

Reply:
![screenshot 2018-05-03 09 03 34](https://user-images.githubusercontent.com/11017722/39563539-e8d6b452-4eb0-11e8-8e2c-ea81e7f8d472.png)

It's on line 162 in this file. For some reason Github didn't allow me to comment on that line directly.